src/test/unit/controller/MultipleRespondentCheckController.test.ts

```diff
@@ -0,0 +1,89 @@
+import MultipleRespondentCheckController from "../../../main/controllers/MultipleRespondentCheckController";
+import sinon from "sinon";
+import { Response } from 'express';
+import { mockRequest } from "../mocks/mockRequest";
+
+const multipleResponseController = new MultipleRespondentCheckController();
+
+
+describe('Mutiple Response Controller Tests', () => {
+    const t = {
+        "multiple-respondent-check": {},
+
+    }
+
+    it(`should render multiple respondent page`, () => {
+        const response = ({ render: () => '' } as unknown) as Response;
```

Comment:
There's a `mockResponse` class that you could reuse instead of duplicating this line.